packages/create-amplify/src/amplify_project_creator.ts

```diff
@@ -46,49 +49,74 @@ export class AmplifyProjectCreator {
 
     await this.packageManagerController.initializeProject();
 
-    await printer.indicateProgress(
-      `Installing required dependencies`,
-      async () => {
-        await this.packageManagerController.installDependencies(
-          this.defaultProdPackages,
-          'prod'
-        );
+    printer.printNewLine();
+    printer.log(format.sectionHeader(`Installing devDependencies:`));
+    printer.log(format.list(this.defaultDevPackages));
+
+    printer.printNewLine();
+    printer.log(format.sectionHeader(`Installing dependencies:`));
+    printer.log(format.list(this.defaultProdPackages));
+    printer.printNewLine();
 
-        await this.packageManagerController.installDependencies(
+    await printer.indicateProgress(
+      () =>
+        this.packageManagerController.installDependencies(
           this.defaultDevPackages,
           'dev'
-        );
-      }
+        ),
+      'Installing devDependencies...',
+      'DevDependencies installed'
```

Comment:
Do we need this as a separate message? If we remove the ... from the installing message we can have `<spinner> Installing devDependencies` => `<check marck> Installing devDependencies` which seems fine to me

Reply:
```
⠹ Installing devDependencies gets replaced with ✔️  DevDependencies installed
```
felt would be minor dx improvement